packages/react-composites/src/composites/ChatComposite/file-sharing/index.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+export * from './FileCard';
+export * from './FileCardGroup';
+export * from './FileUploadButton';
+export * from './FileUploadHandler';
+export * from './FileUpload';
+
+/**
+ * @internal
+ */
+export const FileSharingMetadataKey = 'fileSharingMetadata';
```

Comment:
this isn't used outside this file, does it need exported?

Reply:
Good catch. Will remove the export